docs/figures/example1.cfg

```diff
@@ -1,38 +1,38 @@
 [General]
-  Version         = 3.0
-  CaseDescription = Example 1: Palmerswaard
-  Branch          = Pannerdensch Kanaal & Nederrijn-Lek
-  Reach           = Nederrijn stuwpand Amerongen km  892-922
-  Qthreshold      = 1500.0
-  Ucrit           = 0.3
-  OutputDir       = output
-  Plotting        = True
-  SavePlots       = False
-  FigureDir       = 
-  ClosePlots      = False
-  RiverKM         = 
+  Version          = 3.0
+  CaseDescription  = Example 1: Palmerswaard
+  Branch           = Pannerdensch Kanaal & Nederrijn-Lek
+  Reach            = Nederrijn stuwpand Amerongen km  892-922
+  Qthreshold       = 1500.0
+  Ucrit            = 0.3
+  OutputDir        = output
+  Plotting         = True
+  SavePlots        = False
+  FigureDir        = 
+  ClosePlots       = False
+  RiverKM          = 
 
 [C2]
-  Discharge       = 2000.0
-  Reference       = input\reference\trim-nrf_Q=337m3s-1_map.nc
-  WithMeasure     = input\measure\trim-nrf_Q=337m3s-1_map.nc
+  Discharge        = 2000.0
+  Reference        = input\reference\trim-nrf_Q=337m3s-1_map.nc
+  WithIntervention = input\measure\trim-nrf_Q=337m3s-1_map.nc
 
 [C3]
-  Discharge       = 3000.0
-  Reference       = input\reference\trim-nrf_Q=564m3s-1_map.nc
-  WithMeasure     = input\measure\trim-nrf_Q=564m3s-1_map.nc
+  Discharge        = 3000.0
+  Reference        = input\reference\trim-nrf_Q=564m3s-1_map.nc
+  WithIntervention = input\measure\trim-nrf_Q=564m3s-1_map.nc
```

Comment:
Should the subfolder name be changed from measure to **intervention**?

Reply:
Agreed, but i am not sure where I can retrieve the files from to update te related pictures.

Reply:
changed the names in example1.cfg & example2.cfg, need to update pictures, let discuss how.